packages/style-engine/class-wp-style-engine.php

```diff
@@ -136,6 +136,17 @@ final class WP_Style_Engine {
 				),
 			),
 		),
+		'shadow' => array(
```

Comment:
Let's make this boxShadow, so we can also have textShadow later.

Reply:
We can do this in a different PR

Reply:
Yeh, we should do this in a follow up as `shadow` is used in a number of places that are merged already

Reply:
Actually thinking about it a bit more, the same shadow presets can work for both text and box shadows, so I'm not sure we should rename it...

Reply:
Looks like they don't work. 
[Text shadow](https://www.w3schools.com/cssref/css3_pr_text-shadow.php) doesn't have `spread` prop and supplying a [box-shadow](https://www.w3schools.com/cssref/css3_pr_box-shadow.php) value to text-shadow doesn't work.